srtcore/api.h

```diff
@@ -64,6 +64,8 @@ modified by
 #include "cache.h"
 #include "epoll.h"
 #include "handshake.h"
+#include "core.h"
+
```

Comment:
There's a header file hierarchy order changed here. Now api.h includes core.h, and appropriate types get only header definitions.